packages/rest-specs/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@typespec/rest-specs",
+  "version": "0.37.2",
+  "description": "Spec scenarios and mock apis",
+  "main": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "watch": "tsc -p ./tsconfig.build.json --watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "clean": "rimraf dist/ temp/",
+    "validate-scenarios": "spec validate-scenarios ./http",
+    "generate-scenarios-summary": "spec generate-scenarios-summary ./http",
+    "upload-manifest": "spec upload-manifest ./http --storageAccountName azuresdkcadlranch",
+    "upload-coverage": "spec upload-coverage --storageAccountName azuresdkcadlranch",
+    "validate-mock-apis": "spec validate-mock-apis ./http",
+    "check-scenario-coverage": "spec check-coverage ./http",
+    "serve": "spec serve ./http",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/typespec.git"
+  },
+  "author": "Microsoft",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/microsoft/typespec/issues"
+  },
+  "homepage": "https://github.com/microsoft/typespec#readme",
+  "dependencies": {
+    "@typespec/spec": "workspace:*",
+    "@typespec/spec-api": "workspace:~",
+    "multer": "^1.4.5-lts.1",
+    "axios": "^1.7.5",
+    "dotenv": "~16.4.5"
+  },
+  "devDependencies": {
+    "@azure-tools/typespec-autorest": "0.46.0",
+    "@azure-tools/typespec-azure-resource-manager": "0.46.0",
+    "@azure-tools/typespec-client-generator-core": "0.46.0",
```

Comment:
Remove those 3 dependencies and any specs that depend on it

Reply:
@timotheeguerin  The `@azure-tools/typespec-autorest` and `@azure-tools/typespec-azure-resource-manager` packages have been removed. But, the `@azure-tools/typespec-client-generator-core` package is used in:

1. client
     a. naming
     b. structure
2. encode
     a. bytes
     b. datetime
     c. duration
     d. numeric
3. parameters
     a. basic
     b. body-optionality
     c. collection-format
     d. spread
4. payload
     a. content-negotiation
     b. media-type
     c. xml
5. special-words
6. type
     a. array
     b. dictionary
     c. model
     d. property
           i. additional-properties
           ii. nullable
           iii. optionality
           iv. value-types
     e. scalar
     f. union

Do you really want to remove all of them?

Reply:
Client should go but reference to it should be removed in other packages

Reply:
@timotheeguerin When you say reference should go, do you mean just remove it from the tsp files? Won't that cause issue in generating clients/services from the tsp files?

Reply:
so either its a test that test a specific thing added by azure core or tcgc and in this case that test is in the wrong place and should be deleted anyway or its adding extra annotations that should be 100% optional for unbranded.

Reply:
Thanks for the explanation. I have removed all the references.